internal/app/dbstart/migrate.go

```diff
@@ -31,30 +32,90 @@ func ensureVersionTable(ctx context.Context, db *sql.DB) (int, error) {
 	return version, nil
 }
 
-func getAvailableMigrations(f fs.FS, driver string) ([]int, error) {
+type migrationFile struct {
+	Name    string
+	Version int
```

Comment:
@jules maybe we can have description and driver here too, as we might expand in that direction in the future, just make sure to note against description that it is' there for future purposes. Include a Stringer function for circular verification of the parser.